tests/lib/rules/avoid-leaking-state-in-ember-objects.js

```diff
@@ -66,6 +66,9 @@ eslintTester.run('avoid-leaking-state-in-ember-objects', rule, {
     'export default Foo.extend({ foo: condition ? "foo" : "bar" });',
     'export default Foo.extend({ foo: "foo" && "bar" });',
     'export default Foo.extend({ foo: "foo" || "bar" });',
+    'export default Mixin.create({});',
+    'export default Mixin.create({ harmlessProp: "foo" });',
+    'export default Mixin.create(NestedMixin, {});',
   ],
```

Comment:
Can you add some tests that native classes are ignored for mixins and Ember objects?